guides/_posts/2014-01-20-ccss.html

```diff
@@ -136,14 +136,14 @@
 {% highlight css %}
 
 /* the !require strength ensures this constraint will hold */
-[col-size] == ::window[width] / 12 !require; 
+[col-size] == ::window[width] / 12 !require;
```

Comment:
Use of square brackets here seems inconsistent with other examples I've seen that don't have them.